scripts/remove

```diff
@@ -28,8 +27,5 @@ sudo rm -f /etc/systemd/system/$app.service
 # remove database!!!
 sudo mongo rocketchat --eval "db.dropDatabase()"
 
-# remove user!!!
-sudo userdel -r $serviceuser
```

Comment:
Doesn't that mean we will also get problems when restoring the backup on a clean/new/other system?